annotation/tests/test_jobs.py

```diff
@@ -1,17 +1,41 @@
 import uuid
-from typing import Set, Tuple
-from unittest.mock import MagicMock
+from collections import namedtuple
+from typing import Any, Set, Tuple, Union
+from unittest.mock import MagicMock, call, patch
 from uuid import UUID
 
 import pytest
+from sqlalchemy.orm.attributes import InstrumentedAttribute
 
+from annotation.database import Base
 from annotation.errors import FieldConstraintError
-from annotation.jobs.services import check_annotators, update_inner_job_status
-from annotation.models import Category, File, Job, User
+from annotation.jobs.services import (
```

Comment:
lets import the job service and access it only. otherwise you have so much import here